### ISSUE TITLE
cant run tests if system under test assembly has "UseWPF=true" specified in csproj file

### ISSUE THREAD

**Issue by d79ima** [nunit/nunit-console#1264]
My .net core 3.1 assembly under test has "UseWPF=true" specified in .csproj file.
My test assembly has some tests against some classes in above assembly.
When i run test assembly from nunit3-console, results come back with 0 tests executed.
If i remove "UseWPF=true" and comment out a bunch of code that fails to compile, now i get tests executing correctly from nunit3-console.
Is this expected behavior or a known bug?

**Comment by CharliePoole** [comment]
@d79ima I believe  that your problem is essentially the same as one already fixed in the dev build, except for the fact that that issue had "UseWindowsForms=true." It will help if you can test using a recent 3.16 development build.

If you use chocolatey, you can install centrally using the following command...

```
choco install nunit-console-runner --version 3.16.0-dev00071 --source https://www.myget.org/F/nunit/api/v2
```

For nuget installation into our project, add the above source in visual studio and change your ProjectReference to use the above version.

**Comment by CharliePoole** [comment]
Great, I'll close this issue. You can expect that fix in the 3.16 release, which I'll put out right after the .NET 7.0 final release or by the end of the month, whichever happens first.
